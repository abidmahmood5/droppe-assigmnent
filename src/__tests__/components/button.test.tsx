import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../../components/button';

it('creates a button component', () => {
  const buttonComponent = renderer.create(
    <Button>Add a Product</Button>
  ).toJSON();

  expect(buttonComponent).toMatchSnapshot();
});

