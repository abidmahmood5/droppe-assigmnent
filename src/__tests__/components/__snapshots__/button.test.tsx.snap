// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a button component 1`] = `
<button
  className="button"
>
  Add a Product
</button>
`;
